pom.xml

```diff
@@ -41,7 +41,7 @@
         <dependency>
             <groupId>org.apache.tomcat</groupId>
             <artifactId>tomcat-jasper</artifactId>
-            <version>9.0.12</version>
+            <version>9.0.16</version>
```

Comment:
Whats the purpose to change version?

Reply:
its my configuration, you can keep 9.0.12